graphql/schema/types/performer.graphql

```diff
@@ -33,6 +33,10 @@ type Performer {
   scene_count: Int # Resolver
   scenes: [Scene!]!
   stash_ids: [StashID!]!
+  details: String
+  deathdate: String
+  hair_color: String
+  weight: String
```

Comment:
I think we should consider making this numeric rather than string, so that we can sort by it properly.

We probably also need to consider what unit to use. It may be worth separating this out to another PR if you want to kick this can down the road.